src/plugins/telemetry/public/services/telemetry_sender.ts

```diff
@@ -53,13 +53,22 @@ export class TelemetrySender {
     return false;
   };
 
+  private delayRetrySend = () => {
+    return 60 * (1000 * Math.min(Math.pow(2, this.retryCount), 64)); // 120s, 240s, 480s, 960s, 1920s, 3840s, 3840s, 3840s
```

Comment:
This could be written as `60000 * Math.min(.....)` but writing it this way makes it easier to read at a glance